synapse/rest/client/v1/admin.py

```diff
@@ -63,6 +65,116 @@ def on_GET(self, request, user_id):
         defer.returnValue((200, ret))
 
 
+class UserRegisterServlet(ClientV1RestServlet):
+    PATTERNS = client_path_patterns("/admin/register")
+    SALT_TIMEOUT = 60
+
+    def __init__(self, hs):
+        super(UserRegisterServlet, self).__init__(hs)
+        self.handlers = hs.get_handlers()
+        self.reactor = hs.get_reactor()
+        self.nonces = {}
```

Comment:
can we have a comment for this, please? what does it map to and from?

Reply:
done